builder/docker/artifact_import.go

```diff
@@ -34,12 +34,10 @@ func (a *ImportArtifact) String() string {
 	if tags == nil {
 		return fmt.Sprintf("Imported Docker image: %s", a.Id())
 	}
-	cast := tags.([]interface{})
+	cast := tags.([]string)
```

Comment:
Why was this change needed? 

In looking at the old logic can see the case that the case would fail and would set cast to a nil slice thus skipping the slice. It may help to check that the type assertion did not fail before continuing 

```go
	cast, ok := tags.([]string)
	if !ok {
	  //do something
	  }
```

Reply:
"docker_tags" is a `[]string` slice and it is set here https://github.com/hashicorp/packer/blob/master/post-processor/docker-tag/post-processor.go#L85 
I notice the error because it was crashing to me, so I had to fix it to make it work. 
Definitely will check if it's nill before casting.